whitepapers/tasks/Harvest.md

```diff
@@ -16,13 +16,13 @@
 
 
 ## Parameters
```

Comment:
We can probably condense this down. In hind site this was way to close to actual implementation.